components/02-elements/swatch/_swatch.scss

```diff
@@ -38,47 +57,42 @@ $swatch__border-color--color--active  : $white !default;
         display: flex;
         justify-content: center;
         align-items: center;
-        height: $swatch__width;
-        width: $swatch__width;
-        border: $swatch__border;
-        background: $swatch__background;
-        margin-right: $swatch__margin;
-        margin-bottom: $swatch__margin;
+        min-width: $swatch__option-sizing;
+        min-height: $swatch__option-sizing;
+        margin: $swatch__option-margin;
+        border: $swatch__option-border;
+        background: $swatch__option-background;
+        color: $swatch__option-color;
         transition: $swatch__transition;
-        color: $swatch__label-color;
         cursor: pointer;
 
-        &:hover {
-            border-color: $swatch__border-color-hover;
-            color: $swatch__label-color-hover;
+        &:hover,
+        &:focus {
+            border-color: $swatch__option-border-color-hover;
         }
 
         &:last-child {
             margin-right: 0;
         }
 
-        &--active {
-            background: $swatch__background--active;
-            border-color: $swatch__border-color--active;
-            color: $swatch__label-color--active;
-
-            &:hover {
-                color: $swatch__label-color--active;
-            }
+        @include mq($screen-m) {
+            min-width: $swatch__option-sizing--medium;
+            min-height: $swatch__option-sizing--medium;
```

Comment:
this value is the same as `$swatch__option-sizing` use above, so we can remove it. Btw, please confirm if we shouldn't leave the swatch as a square (48x48) for that medium screen, IMO those 42px vs 48px ot's not the case. Confirm with Vasyl please

Reply:
size swatch should be 48x48 everywhere but colour swatch is not ended yet so I leave it.